qcodes/instrument_drivers/Keysight/keysightb1500/message_builder.py

```diff
@@ -1203,6 +1203,20 @@ def clcorr(self,
                chnum: Union[constants.ChNr, int],
```

Comment:
for all the MessageBuilder methods that are used in this PR, please make sure they are tested in `test_message_builder.py`.

Reply:
Done !